rust/template/differential_datalog/program.rs

```diff
@@ -839,6 +842,28 @@ pub struct RunningProgram<V: Val> {
     pub profile: Arc<Mutex<Profile>>,
 }
 
+// Right now this Debug implementation is more or less a short cut.
+// Ideally we would want to implement Debug for `RelationInstance`, but
```

Comment:
I don't think this is a good comment to describe what this function does. Perhaps the comment should be somewhere else.

Reply:
Thanks for your review. It's not a comment describing what the function does as it's not a doc comment (`///`). I'd say the function is as self-explanatory as it gets. The comment is pointing out why we have a custom implementation of `Debug` and not an auto-derive, which is not obvious from the code from what I can tell.